test/PolylineEditor.js

```diff
@@ -551,6 +551,18 @@ describe('L.PolylineEditor', function() {
             layer.remove();
         });
 
+        it('should fire editable:vertex:new ', function () {
```

Comment:
Would you mind adding one similar test for Polygon? There are subtle differences between PolylineEditor and PolygonEditor, so it's better always checking for both.
Also, one quick test for checking that the event is fired when dragging a middle marker?